### DISCUSSION TITLE
SPI frequency

### DISCUSSION THREAD

**Discussion by chester2400** [Bodmer/TFT_eSPI#2433] [Q&A - General]
I cant seem to change the SPI frequency. I tryed to change it in user_Setup.h
[User_Setup.zip](https://github.com/Bodmer/TFT_eSPI/files/10788126/User_Setup.zip)
[Setup61_RP2040_GC9A01.zip](https://github.com/Bodmer/TFT_eSPI/files/10788129/Setup61_RP2040_GC9A01.zip)
and in user setups for GC9A01.
Whotever value i enter it seems to ignore it and works around 25-27 MHz. 
On teensy I had this screen to work around 35-38MHz with no issues.

**Comment by Bodmer** [comment]
See here:
https://github.com/earlephilhower/arduino-pico/discussions/415
